sda-auth/dev-server/docker-compose.yml

```diff
@@ -48,7 +48,7 @@ services:
     build:
       context: ../
       args:
-        GOLANG_VERSION: ${GOLANG_VERSION-1.18}
+        GOLANG_VERSION: ${GOLANG_VERSION-1.20.7}
```

Comment:
I don't know if using "1.20.7" is over-specifying it.  I noticed that if one pulls the `golang:1.20` image from Dockerhub, one gets Go 1.20.7.

Reply:
there was an error in the tests where the action tried to pull `1.2` instead of `1.20` so I thought that over specifying would fix it but I hadn't noticed that the error was from not having quotes in `go-version: [1.20]`. Anyhow, for documentation reason it's not bad to have this in the compose I suppose.

Reply:
I saw that error too, and I do not understand the cause of it.  I tried to run the build locally but didn't have any problems.  It seemed like the CI tests treated `1.20` as a floating point value for whatever reason and decided to use it without the trailing zero.

Reply:
indeed, adding quotes around `1.20` solved it.

Reply:
Weird.